react-app/src/components/ResultPage.js

```diff
@@ -14,10 +14,18 @@ function ResultPage(props) {
       <Container fluid>
         <Row>
           <Col xxl={3} xl={3} lg={3} md={3} sm={3} xs={3} className="sidebar-col">
-            <ResultSidebar extractedData={props.extractedData} setPatientID={setPatientID} />
+            <ResultSidebar
+              extractedData={props.extractedData}
+              setPatientID={setPatientID}
+              loggedMessages={props.loggedMessages}
```

Comment:
It doesn't look like `ResultsSidebar` is using `props.loggedMessages` at all. Is there any reason to keep it passed into the component? Or can we remove it?

Reply:
Just as a reminder, I think you mentioned you were going to address this comment and the JA1 comment, but I haven't seen those changes get pushed yet. No rush, just wanted to make sure they were still on your radar!

Reply:
Looks like I stashed those changes instead of committing them. I'll push them now.